go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	github.com/speakeasy-api/openapi-generation/v2 v2.257.2
 	github.com/speakeasy-api/openapi-overlay v0.3.0
 	github.com/speakeasy-api/sdk-gen-config v1.7.2
-	github.com/speakeasy-api/speakeasy-client-sdk-go/v3 v3.2.0
-	github.com/speakeasy-api/speakeasy-core v0.1.0
+	github.com/speakeasy-api/speakeasy-client-sdk-go/v3 v3.2.2
+	github.com/speakeasy-api/speakeasy-core v0.1.1-0.20240214203300-eb0fadd7b001
```

Comment:
I'll change this before merging

https://github.com/speakeasy-api/speakeasy-core/pull/16